sdv/_addons.py

```diff
@@ -0,0 +1,24 @@
+"""SDV add-ons functionality."""
+import warnings
+
+from pkg_resources import iter_entry_points
+
+
+def _find_addons(group, parent_globals):
+    """Find and load add-ons based on the given group.
+
+    Args:
+        group (str): The name of the entry points group to load.
+        parent_globals (dict): The caller's global scope. Modules will be added
+            to the parent's global scope through their name.
+    """
```

Comment:
I think this is pretty cool. In some circumstances we may need to add other methods because the grouping might not be as convenient. For example, with transformers, sometimes we want to add them to the `rdt.transformers` globals and sometimes we only want to add them a level in, but for anytime we want to add things at surface level, I think this should work great